server/src/test/java/org/elasticsearch/search/aggregations/AggregatorBaseTests.java

```diff
@@ -80,7 +80,8 @@ private ValuesSourceConfig getVSConfig(
         AggregationContext context
     ) {
         MappedFieldType ft
-            = new NumberFieldMapper.NumberFieldType(fieldName, numType, indexed, false, true, false, null, Collections.emptyMap(), null);
+            = new NumberFieldMapper.NumberFieldType(fieldName, numType, indexed, false, true, false,
+            null, Collections.emptyMap(), null, false);
```

Comment:
I'd hit this line with the formatter. It'll be super big but that's the formatter we've chosen. One day that formatter's going to format everything and when we have to reflow something like this I figure we may as well just do what it's going to do.